docs/README.md

```diff
@@ -0,0 +1,220 @@
+<p align="center"><img width="40%" src="doc/static/allennlp-logo-dark.png" /></p>
```

Comment:
This is duplicating the file at the top level?  Was that removed?  Oh, was this an autogenerated file that you didn't intend to be included at this point?  Same with the other top-level markdown documents.

Reply:
Oh, yes. They are just copied in when they are built, so I can remove them.

Reply:
Yep, I can remove them. They are copied in.